### PR TITLE
topology1: ADL: Add DTS topology

### PR DESCRIPTION
**Pull request by joechengxperi**
Add DTS topology and add a configuration override to enable DTS codec and SOF EQ on ADL

### CODE REVIEW

**Comment by sofci** [comment]
Can one of the admins verify this patch?
> reply **test&nbsp;this&nbsp;please** to run this test once

**Comment by gkbldcig** [comment]
Can one of the admins verify this patch?

**Comment by macchian** [comment]
> Can one of the admins verify this patch?

@joechengxperi Thanks your PR! As we discussed, we need to verify the DTS + DSM + AEC at this moment. Further if having sufficient memory/mcps for others. To be checked.

**Comment by lgirdwood** [comment]
try CI again.

**Comment by lgirdwood** [comment]
please test this.

**Comment by joechengxperi** [comment]
test this please

**Comment by lgirdwood** [comment]
test this please

**Review by markbartonxperi — APPROVED** [review]

File: tools/topology/topology1/CMakeLists.txt [hunk 1]
```diff
@@ -155,6 +155,7 @@ set(TPLGS
 	"sof-tgl-max98373-rt5682\;sof-tgl-rt5682-ssp0-max98373-ssp2-xperi\;-DAMP_SSP=2\;-DINCLUDE_IIR_EQ=1"
 	"sof-adl-nau8825\;sof-adl-nau8825\;-DNO_AMP\;-DBT_OFFLOAD"
 	"sof-adl-nau8825\;sof-adl-max98373-nau8825\;-DAMP_SSP=1\;-DSMART_AMP\;-DBT_OFFLOAD"
+    "sof-adl-nau8825\;sof-adl-max98373-nau8825-dts\;-DAMP_SSP=1\;-DSMART_AMP\;-DBT_OFFLOAD\;-DDTS=`DTS'"
```
Comment (lgirdwood) [inline tools/topology/topology1/CMakeLists.txt:158]: spaces instead of tabs ? (could be GH though rendering incorrectly)

Comment (joechengxperi) [reply tools/topology/topology1/CMakeLists.txt]: It's caused by my Vim setting. Already update it.

**Review by lgirdwood — APPROVED** [review]

**Pull request merged**